libs/designsystem/toast/src/services/toast.helper.ts

```diff
@@ -20,6 +20,10 @@ export class ToastHelper {
       cssClass: this.getCssClass(config.messageType),
     });
     await toast.present();
+    toast.addEventListener('ionToastDidPresent', () => {
```

Comment:
If we do this, consider using the `htmlAttributes` config option to achieve this: https://ionicframework.com/docs/api/toast#htmlattributes

But before that, we should be careful about setting aria-live to be assertive. I think it goes against the intention of the toast being our most subtle way to notify users. 

Ionic directly warns against it: https://ionicframework.com/docs/api/toast#role